src/shared/ui/SubPageHeader/index.tsx

```
@@ -0,0 +1,17 @@
+export interface SubPageHeaderProps {
+  backLink?: string;
+  title: string;
+}
+
+const SubPageHeader: React.FC<SubPageHeaderProps> = ({ backLink, title }) => {
```

ESLint

[error] 'backLink' is defined but never used.

(@typescript-eslint/no-unused-vars)